src/Web/Microsoft.NET.Sdk.Web.ProjectSystem.Targets/netstandard1.0/Microsoft.NET.Sdk.Web.ProjectSystem.targets

```diff
@@ -43,5 +43,14 @@ Copyright (c) .NET Foundation. All rights reserved.
       <Context>BrowseObject</Context>
     </PropertyPageSchema>
   </ItemGroup>
-
+  
+  <!-- Enables IIS settings by copiing non framework assemblies to the bin folder -->
+  <Target Name="CopyAssembliesToOutput" AfterTargets="Build" DependsOnTargets="RunResolvePublishAssemblies" Condition="'$(CreateLocalDeployment)' == 'true'">
+    <Copy SourceFiles="@(ResolvedAssembliesToPublish)" DestinationFiles="$(TargetDir)%(ResolvedAssembliesToPublish.DestinationSubPath)" SkipUnchangedFiles="true" />
```

Comment:
/cc @davidfowl /cc @vijayrkn
I opened the following issue to track getting this functionality into the core SDK: https://github.com/dotnet/sdk/issues/1054